lib/v2/screens/sign_up/viewmodels/states/display_name_state.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:equatable/equatable.dart';
+
+class DisplayNameState extends Equatable {
+  final String? displayName;
+
+  const DisplayNameState({this.displayName});
+
+  @override
+  List<Object?> get props => [displayName];
+
+  DisplayNameState copyWith({
+    String? displayName,
+  }) {
+    return DisplayNameState(
+      displayName: displayName ?? this.displayName,
+    );
+  }
+
+  factory DisplayNameState.initial() => const DisplayNameState(
+        displayName: null,
+      );
+}
```

Comment:
Wasn't it easier just to use a String variable instead a whole model?

Reply:
It would be easier but I wanted to follow the same pattern as other screens in signup flow.